src/views/Correction/FmCorrection.vue

```diff
@@ -28,7 +28,7 @@
 
     <StaffPayment
       class="mt-10"
-      :sectionNumber="isClientErrorCorrection ? '4.' : '2.'"
+      :sectionNumber="isClientErrorCorrection ? '4.' : '3.'"
```

Comment:
I think the previous code was actually correct: if this is not a client error correction then both Completing Party and Certify Section are not rendered, so StaffPayment is indeed the second section.

Reply:
I also wondering why the Certify Section is not displayed here but it IS displayed in a BEN correction? Please confirm with Mihai.

Reply:
You are right. Certify will be displayed for client error, so StaffPayment is the second section. I will change it back in my this PR: https://github.com/bcgov/business-edit-ui/pull/380. I will confirm with Mihai during today's standup and add the condition to Certify if we want to hide Certify for staff error for BEN IA correction.